.circleci/config.yml

```diff
@@ -58,18 +91,50 @@ jobs:
       - *install_goreleaser
       - run: goreleaser
 
+  release_images:
```

Comment:
Instead of re-inventing the wheel and duplicating this code everywhere, perhaps we could use the orb to do this:

https://github.com/FairwindsOps/goldilocks/blob/e41e72bad8161f2437e80b450ed5ebde7092bf86/.circleci/config.yml#L52-L65

Reply:
Funny, I was just thinking, "I bet there's a better way to do this with the orb"